Trait concepts (#135)
exercism/cairo: concepts/traits/introduction.md

# Introduction

> In Cairo, a trait is a series of methods that can be implemented by a data type. When a trait is implemented for a type, the methods defined in that trait can be called on instances of that type. A trait is not limited to only a singular type but when combined with generic type, the functionality defined in the trait can be invoked on a particular type and can share these methods with other data types.

Reviewer: ```suggestion
In Cairo, a trait is a series of methods that can be implemented by a data type. When a trait is implemented for a type, the methods defined in that trait can be called on instances of that type. A trait can be defined with generic types, defining behavior shared by multiple data types.
```
nit: wording
